### PR TITLE
SDI changes

### PR DESCRIPTION
**Pull request by deep-quality-dev**
Changelog:
- Anyone should have the ability to create an issuer.
- The operator or the issuer's creator can update or delete an issuer.
- Rename set-issuer-verification to set-issuer-status.
- Rename set-issuer-details to create-issuer.
- Change the error text "not operator" to "Signer is not operator or issuer creator" for update issuer and remove issuer operations.
- Update unit test.

### CODE REVIEW

**Review by MikkySnow — APPROVED** [review]

File: proto/swisstronik/compliance/genesis.proto [hunk 1]
```diff
@@ -10,15 +10,16 @@ option go_package = "swisstronik/x/compliance/types";
 // GenesisState defines the compliance module's genesis state.
 message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
-  repeated IssuerGenesisAccount issuers = 2;
+  repeated GenesisIssuerDetails issuerDetails = 2;
```
Comment (MikkySnow) [inline proto/swisstronik/compliance/genesis.proto:13]: Looks like we should prepare some upgrade to be able start node with existing genesis.json

File: x/compliance/keeper/msg_server.go [hunk 2]
```diff
@@ -198,26 +194,30 @@ func (k msgServer) HandleUpdateIssuerDetails(goCtx context.Context, msg *types.M
...
+	// Operator or issuer creator can update issuer
+	creator := k.GetIssuerCreator(ctx, issuer)
+	if !signer.Equals(creator) {
+		if exists, err := k.OperatorExists(ctx, signer); !exists || err != nil {
+			return nil, types.ErrNotOperatorOrIssuerCreator
+		}
+	}
+
 	if exists, err := k.IssuerExists(ctx, issuer); !exists || err != nil {
 		return nil, errors.Wrap(types.ErrInvalidIssuer, "issuer not exists")
```
Comment (MikkySnow) [inline x/compliance/keeper/msg_server.go:212]: Please change error text to `issuer does not exist`

File: x/compliance/keeper/msg_server.go [hunk 3]
```diff
@@ -242,16 +242,20 @@ func (k msgServer) HandleRemoveIssuer(goCtx context.Context, msg *types.MsgRemov
...
```
Comment (MikkySnow) [inline x/compliance/keeper/msg_server.go:260]: Replace error text to `issuer does not exist`

**Review by MikkySnow — CHANGES_REQUESTED** [review]

File: proto/swisstronik/compliance/genesis.proto (hunk 1 above)
Comment (deep-quality-dev) [reply proto/swisstronik/compliance/genesis.proto:13]: It doesn't break importing genesis when we migrate to new version.
If there's a migration function for specific module, module manager (module.Mananger) won't call `InitGenesis` within [`RunMigrations`](https://github.com/SigmaGmbH/swisstronik-chain/pull/115/files#diff-1999b5d151142c0e25aa3971448f893be3bcaeca2dd4404fea9ab8d7d4f001a1R16).

**Review by MikkySnow — APPROVED** [review]

**Pull request merged**